src/unicef_security/views.py

```diff
@@ -15,5 +15,9 @@ def get_context_data(self, **kwargs):
 
 class UNICEFLogoutView(RedirectView):
     def get_redirect_url(self, *args, **kwargs):
-        return f'https://{settings.TENANT_B2C_URL}/{settings.TENANT_ID}/{settings.POLICY}/oauth2/v2.0/' \
+
+        base_url = getattr(settings, 'TENANT_B2C_URL', 'login.microsoftonline.com')
+        tenant_id = getattr(settings, 'SOCIAL_AUTH_TENANT_ID', settings.TENANT_ID)
+
+        return f'https://{base_url}/{tenant_id}/oauth2/v2.0/' \
```

Comment:
is the policy no longer required? /  will this not break things in other places?

Reply:
looks is not

Reply:
if it's all the same, I'd like to keep policy in. I'm not super-familiar with the inner workings of AD logout, but in my experience, being more specific is usually a good thing. (From the AD docs is where I've picked up that policy that is templated in the URL) - Does it break functionality in your app if we keep it in?